pkg/security/probe/probe_windows.go

```diff
@@ -45,16 +52,111 @@ type WindowsProbe struct {
+					<keyword name="KERNEL_FILE_KEYWORD_WRITE" message="$(string.keyword_KERNEL_FILE_KEYWORD_WRITE)" mask="0x200"/>
+					<keyword name="KERNEL_FILE_KEYWORD_DELETE_PATH" message="$(string.keyword_KERNEL_FILE_KEYWORD_DELETE_PATH)" mask="0x400"/>
+					<keyword name="KERNEL_FILE_KEYWORD_RENAME_SETLINK_PATH" message="$(string.keyword_KERNEL_FILE_KEYWORD_RENAME_SETLINK_PATH)" mask="0x800"/>
+					<keyword name="KERNEL_FILE_KEYWORD_CREATE_NEW_FILE" message="$(string.keyword_KERNEL_FILE_KEYWORD_CREATE_NEW_FILE)" mask="0x1000"/>
+		    	</keywords>
+		*/
+		// try masking on create & create_new_file
+		// given the current requirements, I think we can _probably_ just do create_new_file
+		cfg.MatchAnyKeyword = 0x10A0
+	})
+	p.fimSession.ConfigureProvider(p.regguid, func(cfg *etw.ProviderConfiguration) {
+		cfg.TraceLevel = etw.TRACE_LEVEL_VERBOSE
+		cfg.PIDs = pidsList
+
+		// full manifest is here https://github.com/repnz/etw-providers-docs/blob/master/Manifests-Win10-17134/Microsoft-Windows-Kernel-Registry.xml
+		/* the mask keywords available are
+				 <keywords>
+					<keyword name="CloseKey" message="$(string.keyword_CloseKey)" mask="0x1"/>
+					<keyword name="QuerySecurityKey" message="$(string.keyword_QuerySecurityKey)" mask="0x2"/>
+					<keyword name="SetSecurityKey" message="$(string.keyword_SetSecurityKey)" mask="0x4"/>
+					<keyword name="EnumerateValueKey" message="$(string.keyword_EnumerateValueKey)" mask="0x10"/>
+					<keyword name="QueryMultipleValueKey" message="$(string.keyword_QueryMultipleValueKey)" mask="0x20"/>
+					<keyword name="SetInformationKey" message="$(string.keyword_SetInformationKey)" mask="0x40"/>
+					<keyword name="FlushKey" message="$(string.keyword_FlushKey)" mask="0x80"/>
+					<keyword name="SetValueKey" message="$(string.keyword_SetValueKey)" mask="0x100"/>
+					<keyword name="DeleteValueKey" message="$(string.keyword_DeleteValueKey)" mask="0x200"/>
+					<keyword name="QueryValueKey" message="$(string.keyword_QueryValueKey)" mask="0x400"/>
+					<keyword name="EnumerateKey" message="$(string.keyword_EnumerateKey)" mask="0x800"/>
+					<keyword name="CreateKey" message="$(string.keyword_CreateKey)" mask="0x1000"/>
+					<keyword name="OpenKey" message="$(string.keyword_OpenKey)" mask="0x2000"/>
+					<keyword name="DeleteKey" message="$(string.keyword_DeleteKey)" mask="0x4000"/>
+					<keyword name="QueryKey" message="$(string.keyword_QueryKey)" mask="0x8000"/>
+		    	</keywords>
+		*/
+		// try masking on create & create_new_file
+		// given the current requirements, I think we can _probably_ just do create_new_file
+		cfg.MatchAnyKeyword = 0xF7E3
+	})
+
+	err = p.fimSession.EnableProvider(p.fileguid)
+	if err != nil {
+		log.Warnf("Error enabling provider %v", err)
+		return err
+	}
+	err = p.fimSession.EnableProvider(p.regguid)
+	if err != nil {
+		log.Warnf("Error enabling provider %v", err)
+		return err
+	}
```

Comment:
I don't think so.  Since this is for CWS (security).  If we can't get some of the information we deem important, IMO we should completely fail the initialization.